toonz/sources/toonz/viewerpane.cpp

```diff
@@ -588,6 +600,12 @@ void SceneViewerPanel::onSceneChanged() {
   TApp *app         = TApp::instance();
   ToonzScene *scene = app->getCurrentScene()->getScene();
   assert(scene);
+  m_flipConsole->setFrameRate(TApp::instance()
+                                  ->getCurrentScene()
+                                  ->getScene()
+                                  ->getProperties()
+                                  ->getOutputProperties()
+                                  ->getFrameRate());
```

Comment:
Oh, I see. I agree that the console fps should be synchronized with the scene settings.
However, I think the value should not be set back every moment you change the scene... 
Consider that you are setting the scene fps to 24, if you temporary set console fps to 8 (for checking a movement), you might not want to set it back to 24 once you edit the xsheet.
Well, let me think.